plugins/vendor/zoom/ZoomDropFolderPlugin/batch/KZoomDropFolderEngine.php

```diff
@@ -215,7 +227,7 @@ protected function getEntryByReferenceId($referenceId)
 	{
 		$entryFilter = new KalturaBaseEntryFilter();
 		$entryFilter->referenceIdEqual = $referenceId;
-		$entryFilter->updatedAtGreaterThanOrEqual = time() - (self::HOUR * 3);
+		$entryFilter->updatedAtGreaterThanOrEqual = time() - (self::DEFAULT_ZOOM_QUERY_TIMERANGE);

```

Comment:
you dont need () for DEFAULT_ZOOM_QUERY_TIMERANGE

Reply:
true, thanks